.babelrc

```diff
@@ -1,10 +1,7 @@
 {
   "env": {
```

Comment:
Were you able to test jest with this config? I'm not sure if you need "env" as part of your obj.